### PR TITLE
updated README with project boards and UX info

### PR DESCRIPTION
**Pull request by pietrop**
**Is your Pull Request request related to another [issue](https://github.com/bbc/react-transcript-editor/issues) in this repository ?**      

NA

**Describe what the PR does**    

Updated the `README` with info and links around the project boards being used

**State whether the PR is ready for review or whether it needs extra work**    

Ready for review

**Additional context**    

Should the mention around UX being added separately from `README`, eg in the `docs/guides`?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,30 @@ See [modular architecture section](./docs/guides/modular-architecture.md) for mo
...
+
+The project is divided into [concurrent milestones as described here](https://github.com/bbc/digital-paper-edit-client/milestones)
+
+And there are [github project board for each milestone](https://github.com/bbc/digital-paper-edit-client/projects)
+
+- [1 - Transcript - Editing](https://github.com/bbc/digital-paper-edit-client/projects/2)
+- [2 - Transcript - Annotating](https://github.com/bbc/digital-paper-edit-client/projects/3)
+- [3 - Paper-editing](https://github.com/bbc/digital-paper-edit-client/projects/4)
+
+With UX being an overarching thing that cuts across these different parts
```
Comment (allishultes) [inline README.md]: This is a picky wordy thing - but UX is its own 'milestone'? (So has a board like the other milestones?) 'Thing' is a bit vague. Workstream could be a better word if it's not a milestone.

File: README.md [hunk 2]
```diff
@@ -30,6 +30,30 @@ See [modular architecture section](./docs/guides/modular-architecture.md) for mo
...
+- [UX](https://github.com/bbc/digital-paper-edit-client/projects/1)
+
+
+## UX
```
Comment (allishultes) [inline README.md]: Yeah I agree with your comment that I'm not sure this section belongs here. I'd move to Confluence :)

File: README.md [hunk 3]
```diff
@@ -30,6 +30,30 @@ See [modular architecture section](./docs/guides/modular-architecture.md) for mo
 
 The projects use [npm semantic versioning](https://docs.npmjs.com/about-semantic-versioning)
 
+## Project boards
+
+Between the 5th of June and 17th of July, there is a sprint board used for organising the backlog for each two weeks sprint cycle.
```
Comment (allishultes) [inline README.md]: Super picky date formatting thing - BBC standard is 5 June and 17 July. Just think it's cleaner but it's no biggie :) and maybe change two weeks to two-week.

File: README.md [hunk 4]
```diff
@@ -30,6 +30,30 @@ See [modular architecture section](./docs/guides/modular-architecture.md) for mo
...
+
+- [BBC News Labs - Digital Paper Edit - Sprint Board](https://github.com/orgs/bbc/projects/33) is a github project board [with linked repository](https://help.github.com/en/articles/linking-a-repository-to-a-project-board) where issues are added from the relevant repository listed in architecture section.
```
Comment (allishultes) [inline README.md]: Again super picky but you might want to do the capitals for GitHub :) Up to you. It's clear either way.

File: README.md (hunk 1 above)
Comment (pietrop) [reply README.md]: thanks, going for `milestone`

**Comment by pietrop** [comment]
Thanks @allishultes sorry for the delay in getting back to this changes.
- [x] I've applied your suggestions
- [x] moved the UX section in a separate file inside the docs linked form the README

File: README.md (hunk 2 above)
Comment (pietrop) [reply README.md]: ok, moved it to separate section in docs

**Pull request merged**